app/src/main/java/com/team41/boromi/auth/RecoverPasswordFragment.java

```diff
@@ -0,0 +1,65 @@
+package com.team41.boromi.auth;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.team41.boromi.R;
+
+/**
+ * A simple {@link Fragment} subclass. Use the {@link RecoverPasswordFragment#newInstance} factory
+ * method to create an instance of this fragment.
+ */
+public class RecoverPasswordFragment extends Fragment {
+
+  // TODO: Rename parameter arguments, choose names that match
+  // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+  private static final String ARG_PARAM1 = "param1";
+  private static final String ARG_PARAM2 = "param2";
+
+  // TODO: Rename and change types of parameters
+  private String mParam1;
+  private String mParam2;
+
+  public RecoverPasswordFragment() {
+    // Required empty public constructor
+  }
+
+  /**
+   * Use this factory method to create a new instance of this fragment using the provided
+   * parameters.
+   *
+   * @param param1 Parameter 1.
+   * @param param2 Parameter 2.
+   * @return A new instance of fragment RecoverPassword.
+   */
+  // TODO: Rename and change types and number of parameters
+  public static RecoverPasswordFragment newInstance(String param1, String param2) {
+    RecoverPasswordFragment fragment = new RecoverPasswordFragment();
+    Bundle args = new Bundle();
+    args.putString(ARG_PARAM1, param1);
+    args.putString(ARG_PARAM2, param2);
+    fragment.setArguments(args);
+    return fragment;
+  }
+
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    if (getArguments() != null) {
+      mParam1 = getArguments().getString(ARG_PARAM1);
+      mParam2 = getArguments().getString(ARG_PARAM2);
+    }
+  }
+
+  @Override
+  public View onCreateView(LayoutInflater inflater, ViewGroup container,
+      Bundle savedInstanceState) {
+    // Inflate the layout for this fragment
+    return inflater.inflate(R.layout.fragment_recover_password, container, false);
+  }
+}
```

Comment:
newline

Reply:
For the newline issue, that's actually an autoformatting thing from google :'(